### ISSUE TITLE
I followed the guide but does not working in my env

### ISSUE THREAD

**Issue by Himly1** [OptimalBits/bull#2424]
## Description
I followed the guide but after I finished my work the code did not  response as I expected.
Below  is the image about the code and the output
 
![Screenshot from 2022-09-16 18-19-29](https://user-images.githubusercontent.com/25456846/190618766-92a2f0cf-cae5-4f9f-a5af-2c42eb2849e6.png)

You can see the output is stuck at the function `add` it is because i put the `await` at the beginning of the function call
If I remove the `await` statement the function will not stuck but the callback function of the  `process` will not output anything which mean it is not calling the callback function or there is no job in the queue

here is the pic of the output of the code which removed the `await` statement
![Screenshot from 2022-09-16 18-32-04](https://user-images.githubusercontent.com/25456846/190619800-9c874e90-f6ab-4299-85a8-6e5b19610e57.png)
but still the  output and the debugger tell me the callback function are not triggered... I dont know what happend, is no job in the queue or something else happend ?

#### (An easy to reproduce test case will dramatically decrease the resolution time.)
here is the script 
[script.zip](https://github.com/OptimalBits/bull/files/9583280/script.zip)

## Bull version
^4.9.0
## Additional information
`Node version`
v16.16.0
`Operation system`
Linux (Manjaro)

**Comment by manast** [comment]
Which guide did you follow? the code above is not correct, for example the ```Queue.add``` call is not passing the correct arguments.

**Comment by Himly1** [comment]
@manast  Thanks for your reply. I found it somewhere else but how about this one ?
![bug](https://user-images.githubusercontent.com/25456846/190648565-9ce5f43c-c33b-43c9-9698-2bbed06bb7f2.png)

I copied that code from the README file of the BULL and modified a little bit but the result is the same on my laptop.
![image](https://user-images.githubusercontent.com/25456846/190648959-e1c0a0e0-96fb-489d-8ca8-7170bfb6f543.png)

**Comment by manast** [comment]
Do you have a Redis instance running? Maybe you should start with BullMQ instead: docs.bullmq.io there are also complete tutorials here: blog.taskforce.sh

**Comment by Himly1** [comment]
@manast  No, did not have a redis instance running. I will check it out . thanks.

**Comment by Himly1** [comment]
@manast  After I installed the redis I working good, Thank you a lot...
